modular_skyrat/code/modules/research/designs/AI_module_designs.dm

```diff
@@ -6,3 +6,30 @@
 	build_path = /obj/item/aiModule/core/full/father
 	category = list("AI Modules")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/ranger_module
```

Comment:
whats the point if you dont add them to a disk/node?

Reply:
Aaaaaaaaaaand my bad. I'll fix that.

Reply:
Wait. They're already in a node...?